weave/trace_server/trace_server_interface.py

```diff
@@ -147,11 +147,89 @@ class _CallsFilter(BaseModel):
     wb_run_ids: typing.Optional[typing.List[str]] = None
 
 
+# Can be any standard json-able value
```

Comment:
Outstanding design decision:
* what to call `filter_by` in the API
* whether or not to remove the nested `filter` property in the `_FilterBy` type
* where to put `filter_by` - should it be moved into the filter?

Once this final decision is made, we need to update the types and usages across interfaces, tests, and application.